fe/fe-core/src/test/java/org/apache/doris/planner/QueryPlanTest.java

```diff
@@ -1360,6 +1376,32 @@ public void testAggregateSatisfyOlapTableDistribution() throws Exception {
         explainString = UtFrameUtils.getSQLPlanOrErrorMsg(connectContext, "EXPLAIN " + sql);
         System.out.println(explainString);
         Assert.assertTrue(explainString.contains("AGGREGATE (update serialize)"));
+    
+        // 测试colocate join是否能够colocate agg
```

Comment:
Thanks for your PR, but please using english to comment

Reply:
fixed

Reply:
@morningman  Hi~ see this pr if have free time～